### PR TITLE
[frontend] Add technical creator in data + ordering/filtering (#2029)

### LINKED ISSUE #2029: Add technical creator in data + ordering/filtering
Browse to /dashboard/data/

Realize that some data is impossible to trace back to the connector that generated it. The Author -column sometimes gives a hint, but many times is empty.

Suggested feature: add a column with the name of connector that ingested the data.

### PR DESCRIPTION
**Pull request by SamuelHassine**
### Proposed changes

* [frontend] Add a column creator in the data entities section (#2029)

### Related issues

* #2029

### CODE REVIEW

File: opencti-platform/opencti-front/src/components/ExpandableMarkdown.js [hunk 1]
```diff
@@ -43,13 +43,10 @@ export const MarkDownComponents = (theme) => ({
 });
 
 const ExpandableMarkdown = (props) => {
+  const { source, limit, theme } = props;
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/components/ExpandableMarkdown.js]: Should be destructure in function declaration :)

File: opencti-platform/opencti-front/src/private/components/data/entities/EntitiesStixDomainObjectLine.js [hunk 2]
```diff
@@ -48,133 +47,119 @@ const styles = (theme) => ({
...
-    } = this.props;
-    const link = `${resolveLink(node.entity_type)}/${node.id}`;
-    return (
-      <ListItem
-        classes={{ root: classes.item }}
-        divider={true}
-        button={true}
-        onClick={onToggleEntity.bind(this, node)}
-        selected={node.id in (selectedElements || {})}
+const EntitiesStixDomainObjectLineComponent = (props) => {
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/data/entities/EntitiesStixDomainObjectLine.js]: Same

File: opencti-platform/opencti-front/src/private/components/data/entities/EntitiesStixDomainObjectLine.js [hunk 3]
```diff
@@ -314,126 +318,126 @@ const EntitiesStixDomainObjectLineFragment = createFragmentContainer(
...
 
-class EntitiesStixDomainObjectLineDummyComponent extends Component {
-  render() {
-    const { classes, dataColumns } = this.props;
-    return (
-      <ListItem
-        classes={{ root: classes.item }}
-        divider={true}
+export const EntitiesStixDomainObjectLineDummy = (props) => {
+  const { dataColumns } = props;
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/data/entities/EntitiesStixDomainObjectLine.js]: Same

File: opencti-platform/opencti-front/src/private/components/common/lists/Filters.js [hunk 4]
```diff
@@ -1691,63 +1714,58 @@ class Filters extends Component {
       R.sortWith([R.ascend(R.prop('label'))]),
     )(entityTypes);
     return (
-      <React.Fragment>
```
Comment (RomuDeuxfois) [inline opencti-platform/opencti-front/src/private/components/common/lists/Filters.js:1720]: Si je comprends bien, on supprime React.Fragment puisqu'on a qu'un seul enfant imbriqué ?

Comment (richard-julien) [reply opencti-platform/opencti-front/src/private/components/common/lists/Filters.js:1720]: Yes, react fragment is just a way to wrap multi component without extra div.
So if only one, its useless

**Comment by richard-julien** [comment]
New way to handling this problem
- Technical creator will be written in the element at creation
- For old elements, a background task is available to Replace "Technical Creator" from history
- Colum for entities/relations with Technical creator is added.
- Filtering for a specific technical creator is now possible

**Pull request merged**